docs/zkEmailDocs/UsageGuide/README.md

```diff
@@ -143,15 +145,11 @@ node --max-old-space-size=614400 ./../node_modules/.bin/snarkjs
 
 
 ### Powers of Tau
-After obtaining the constraint size, find the next highest power of 2 and replace the '12' in the following command with that number. This command initiates the Powers of tau ceremony.
 
-```
-snarkjs powersoftau new bn128 12 pot12_0000.ptau -v
-```
+You can download the ptau file directly from Google Cloud Platform using the following command:
 
-Then contribute to the ceremony by running:
-```bash
-snarkjs powersoftau contribute pot12_0000.ptau pot12_0001.ptau --name="First contribution" -v
+```
+wget https://storage.googleapis.com/zkevm/ptau/powersOfTau28_hez_final_22.ptau
```

Comment:
The file they want to download would depend on the constraints the final circuit will have. Would be good to add this info.
The mapping is here - https://github.com/iden3/snarkjs?tab=readme-ov-file#7-prepare-phase-2
We  can provide links for 21,22,23 I think